api/src/main/java/org/openmrs/module/attachments/obs/Attachment.java

```diff
@@ -48,6 +54,29 @@ public Attachment(Obs obs) {
 		setComplexData(obs.getComplexData());
 	}
 	
+	/**
+	 * @param obs A complex obs
+	 */
+	public Attachment(Obs obs, ComplexDataHelper complexDataHelper) {
+		super();
```

Comment:
Replace this with 

`this(obs)`

and remove the calls to the setters already called in `Attachment(Obs)`

Reply:
@ivange94 its done and the tests are passing now

Reply:
Thank you @gitcliff

Reply:
@gitcliff can you put this PR to ready for review? It's currently a draft.